crates/re_renderer/shader/outlines/jumpflooding_init.wgsl

```diff
@@ -0,0 +1,145 @@
+        num_edges_and_b += has_edge;
+        edge_pos_a_and_b += has_edge.xxyy * 0.5;
+    }
+
+    // A lot of this code is repetetive, but wgsl/naga doesn't know yet how to do static indexing from unrolled loops.
+
+    // Sample closest neighbors top/bottom/left/right
+    { // right
+        let has_edge = has_edge(mask_right_top, center_coord + IVec2(1, 0), 2);
+        let edge_pos = Vec2(1.0, 0.5);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // bottom
+        let has_edge = has_edge(mask_bottom_right, center_coord + IVec2(0, 1), 0);
+        let edge_pos = Vec2(0.5, 1.0);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // left
+        let has_edge = has_edge(mask_left_bottom, center_coord + IVec2(-1, 0), 1);
+        let edge_pos = Vec2(0.0, 0.5);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // top
+        let has_edge = has_edge(mask_top_left, center_coord + IVec2(0, -1), 3);
+        let edge_pos = Vec2(0.5, 0.0);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+
+    // Sample closest neighbors diagonally.
+    // This is not strictly necessary, but empirically the result looks a lot better!
+    { // top-right
+        let has_edge = has_edge(mask_right_top, center_coord + IVec2(1, -1), 2);
+        let edge_pos = Vec2(1.0, 0.0);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // bottom-right
+        let has_edge = has_edge(mask_bottom_right, center_coord + IVec2(1, 1), 0);
+        let edge_pos = Vec2(1.0, 1.0);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // bottom-left
+        let has_edge = has_edge(mask_left_bottom, center_coord + IVec2(-1, 1), 1);
+        let edge_pos = Vec2(0.0, 1.0);
+        edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+    { // top-left
+        let has_edge = has_edge(mask_top_left, center_coord + IVec2(-1, -1), 3);
+        //let edge_pos = Vec2(0.0, 0.0);
+        //edge_pos_a_and_b += Vec4(edge_pos, edge_pos) * has_edge.xxyy;
+        num_edges_and_b += has_edge;
+    }
+
+    // Normalize edges ans get range from [0, 1] to [-0.5, 0.5].
+    edge_pos_a_and_b = edge_pos_a_and_b / num_edges_and_b.xxyy - Vec4(0.5);
+
+    var pixel_coord_a: Vec2;
+    if num_edges_and_b.x == 0.0 {
+        pixel_coord_a = Vec2(inf());
+    } else {
+        pixel_coord_a = Vec2(center_coord) + edge_pos_a_and_b.xy;
+    }
+    var pixel_coord_b: Vec2;
+    if num_edges_and_b.y == 0.0 {
+        pixel_coord_b = Vec2(inf());
+    } else {
+        pixel_coord_b = Vec2(center_coord) + edge_pos_a_and_b.zw;
+    }
```

Comment:
You've explained it to me, but it'd be nice to have the reason for the switch to pixel coordinates documented here!